packages/PySimpleGUI.yml

```diff
@@ -2,5 +2,5 @@ repo: PySimpleGUI/PySimpleGUI
 section: gui applications
 description: Create custom GUIs quickly & easily on top of tkinter, Qt, WxPython or Remi
 site: http://www.PySimpleGUI.com
-keywords:  GUI tkinter pyside2 WxPython Remi documentation cookbook Trinkets replit
-pypi_name:  PySimpleGUI
+keywords: [GUI, tkinter, pyside2, WxPython, Remi, documentation, cookbook, Trinkets, replit]
```

Comment:
Why have these all been put in square brackets?

Reply:
So that they will parse into a Python list rather than a string containing commas.
![image](https://user-images.githubusercontent.com/1695496/146589826-93494936-4dd4-4541-ab5a-b41bc8ae733b.png)

Reply:
Keywords aren't currently used, but surely if we did use them it would be as a list rather than as a single string.

Reply:
Yes but I'd rather we took that burden, rather than failing if someone didn't use brackets.  Splitting a string on commas is pretty easy.

Reply:
Up to you, though since you control this repo, seems easy enough if keywords do get used someday to fix any broken entries. And parsing multiword items in lists the fairly reasonable way YAML does is not trivial:

![image](https://user-images.githubusercontent.com/1695496/146602909-954affff-8d01-4e0d-b983-d8ca7b539457.png)

Reply:
I think since @jbedner already went through the work of converting everything to lists, is not much work to require it going forward (especially if we update the readme w/ an example

Reply:
I did update the readme in that way, btw.

Reply:
+1 for standard YAML syntax.